install_script.sh.template

```diff
@@ -1122,6 +1127,10 @@ EOF
       formatted_host_tags="['""$( echo "$host_tags" | sed "s/,/','/g" )""']"  # format `env:prod,foo:bar` to yaml-compliant `['env:prod','foo:bar']`
       $sudo_cmd sh -c "sed -i \"s/^# tags:.*/tags: ""$formatted_host_tags""/\" $config_file"
   fi
+  if [ "$dd_env" ]; then
+    printf "\033[34m\n* Adding your DD_ENV to the $nice_flavor configuration: $config_file\n\033[0m\n"
+    $sudo_cmd sh -c "sed -i 's/^# env:.*/env: $dd_env/' $config_file"
```

Comment:
not sure if we can do this
```suggestion
    $sudo_cmd sh -c "sed -i '0,/^# env/{s/^# env:.*/env: $dd_env/}' $config_file"
```
if we know `env` is supposed to be the first in config file it could help make the replacement more robust (be sure we don't do spare replacements)

Reply:
That seems sensible, but it would also imply that the yaml has duplicated keys and is therefor invalid no?

Reply:
yes it's risky, but for some replacements (e.g. tags) where we had issues, it could have  prevented errors